### PR TITLE
Fix summary style for grouped household style answers

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
- Fixes answer alignment for grouped "household" style answers
- Adds these type of answers into one of the examples
- Adds some spacing to answers that come after "household" style answers
- Removes headers param from summary test because the param is no longer used

Fixes: #2510 

### How to review
- Check that all types of summary still work and look as expected
- Check that summary answers for "household" style answers are now aligned at all screen sizes

### CODE REVIEW

**Comment by rmccar** [comment]
[Fix alignment in DS for grouped list collector answers](https://trello.com/c/PlhYTpdF/5481-fix-alignment-in-ds-for-grouped-list-collector-answers)

File: src/components/summary/_macro.njk [hunk 1]
```diff
@@ -112,7 +112,7 @@
                     {% endif %}
 
                     {% if group.summaryLink %}
-                        <div class="ons-summary__link{% if group.placeholderText or group.rows %} ons-u-pt-s{% endif %}{% if group.placeholderText is not defined and group.rows | length > 1 %} ons-u-bt{% endif %}">
+                        <div class="ons-summary__link{% if group.placeholderText or group.rows %} ons-u-pt-s{% endif %}{% if group.placeholderText is not defined and group.rows | length > 1 %} ons-u-bt{% endif %}{% if not group.last %} ons-u-mb-xl{% endif %}">
```
Comment (jrbarnes9) [inline src/components/summary/_macro.njk:115]: Should probably add a test to `_macro.spec.js` for the addition of this class

**Review by jrbarnes9 — COMMENTED** [review]
Actions not right-aligned in Safari...
Screenshot 2022-11-30 at 10 58 37

**Comment by rmccar** [comment]
> Actions not right-aligned in Safari... Screenshot 2022-11-30 at 10 58 37

I noticed this in Percy myself but I cant recreate it locally in Safari, is it possible that Percy is using an older version?
It also seems to be an issue on the `main` branch and not just on my PR branch so is probably its own issue out of scope of this PR

**Pull request merged**